Disable ALSA backend
Leave Rust sources to devel/cargo module

Index: src/meson.build
--- src/meson.build.orig
+++ src/meson.build
@@ -9,7 +9,6 @@ dependency('libadwaita-1')
 dependency('gtk4')
 dependency('glib-2.0')
 dependency('openssl')
-dependency('alsa')
 dependency('libpulse')
 
 conf = configuration_data()
@@ -85,6 +84,7 @@ run_command(
   check: true
 )
 
+subdir_done()
 cargo = find_program('cargo', required: true)
 
 if get_option('buildtype') == 'release'
